package.json

```diff
@@ -2,8 +2,8 @@
   "name": "hkm",
   "version": "0.0.1",
   "scripts": {
-    "browser-test": "testcafe \"chrome --window-size='1920,1080'\" browser-tests/enable-dotenv.js browser-tests/ --live",
-    "browser-test:ci": "testcafe \"chrome:headless --disable-gpu --window-size='1920,1080'\" --screenshots path=report takeOnFails=true --video report browser-tests/"
+    "browser-test": "testcafe \"chrome --lang=fi-FI --window-size='1920,1080'\" browser-tests/enable-dotenv.js browser-tests/ --live",
```

Comment:
Why are these 2 using a different language? Shouldn't they be exactly the same and only difference be that another one is ran in a non graphical environment.

Reply:
This is actually a mistake. Forcing the UI language is necessary to make sure test are run in the language (English) that the tests assume. But the ci configuration had no effect in the ci-env and the local language was there just to simulate similar failure than in ci-env. Will remove both. Forcing English as the UI language is done using the app feature to set the language before page load.